frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeAddTemplate.tsx

```diff
@@ -101,40 +123,47 @@ const CustomServingRuntimeAddTemplate: React.FC<CustomServingRuntimeAddTemplateP
       empty={false}
       provideChildrenPadding
     >
-      <Stack hasGutter>
-        <StackItem isFilled>
-          <DashboardCodeEditor
-            code={code}
-            isUploadEnabled
-            isLanguageLabelVisible
-            language={Language.yaml}
-            height="100%"
-            options={{ tabSize: 2 }}
-            emptyStateTitle="Add a serving runtime"
-            emptyStateBody="Drag a file here, upload files, or start from scratch."
-            emptyStateButton="Upload files"
-            onCodeChange={(codeChanged: string) => {
-              setCode(codeChanged);
-            }}
-          />
-        </StackItem>
-        {error && (
+      <Form style={{ height: '100%' }}>
+        <Stack hasGutter>
           <StackItem>
-            <Alert
-              isInline
-              variant="danger"
-              title={error.name}
-              actionClose={<AlertActionCloseButton onClose={() => setError(undefined)} />}
-            >
-              {error.message}
-            </Alert>
+            <CustomServingRuntimePlatformsSelector
```

Comment:
Mmmmmm I'm trying to follow the logic, but somehow when clonning the Serving Runtime it doesn't update the platform selected